installers/cardano-installer.cabal

```diff
@@ -15,23 +15,33 @@ executable make-installer
   other-modules:       MacInstaller
                      , WindowsInstaller
                      , RewriteLibs
+
   build-depends:       base
+                     , bytestring
+                     , containers
+                     , dhall
+                     , dhall-json
                      , directory
                      , filepath
                      , Glob
                      , megaparsec
                      , nsis
+                     , optparse-applicative
+                     , optparse-generic
+                     , optional-args
                      , split
                      , system-filepath
                      , temporary
                      , temporary
                      , text
+                     , trifecta
```

Comment:
Do we really need trifecta now that we use dhall-json as a lib?

Reply:
Right, good catch! : -)